cspell.json

```diff
@@ -64,19 +64,32 @@
   "words": [
     "apikey",
     "buildhome",
+    "bodycol",
     "callout",
     "checkmark",
+    "closebutton",
+    "closebtncol",
     "coachmark",
+    "coachmarks",
+    "Coachmarks",
+    "COACHMARKSTACKHOME",
     "colindex",
     "columnheader",
+    "Cpath",
+    "Csvg",
```

Comment:
You might be able to rename these in the svg files but if we have control over this naming, we can probably call it out as just `coachmarkPath` or `coachmarkSvg`? If we can’t then at least we should keep this lowercase as `cpath`, `csvg`.

Reply:
Yep, so this one is kindof a silly one... its at a part within an SVG where we are URL encoding something that looks like this `><path` - where that url encoded value is `%3E%3Cpath`... I will just URL decode that and remove it from cspell